### PR TITLE
Synopsysctl help clean up

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/388#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/388#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/388#pullrequestreview-243932378" title="Approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/alert/ctl_alert.go [hunk 1]
```diff
@@ -167,8 +152,9 @@ func (ctl *Ctl) SetFlag(f *pflag.Flag) {
 		case "cfssl-image":
 			ctl.Spec.CfsslImage = ctl.CfsslImage
 		case "stand-alone":
-			ctl.Spec.StandAlone = &ctl.StandAlone
-		case "expose-service":
+			standAloneVal := strings.ToUpper(ctl.StandAlone) == "TRUE"
```
Comment (yashbhutwala) [inline pkg/alert/ctl_alert.go]: @mphammer I wonder if use of [`strconv.ParseBool`](https://golang.org/pkg/strconv/#ParseBool) would be better here.

File: pkg/synopsysctl/cmd_deploy.go [hunk 2]
```diff
@@ -115,15 +116,15 @@ var deployCmd = &cobra.Command{
 			return nil
 		}
 
-		// Deploy synopsys-operator
-		log.Debugf("creating Synopsys-Operator components")
+		// Deploy Synopsys Operator
+		log.Debugf("creating Synopsys Operator components")
 		soperatorSpec := soperator.NewSOperator(deployNamespace, synopsysOperatorImage, exposeUI, adminPassword, postgresPassword,
-			userPassword, blackduckPassword, secretType, operatorTimeBombInSeconds, dryRun, logLevel, threadiness, postgresRestartInMins,
+			userPassword, blackDuckPassword, secretType, operatorTimeBombInSeconds, strings.ToUpper(dryRun) == "TRUE", logLevel, threadiness, postgresRestartInMins,
```
Comment (yashbhutwala) [inline pkg/synopsysctl/cmd_deploy.go]: Also, this check looks pretty ugly in passing of parameters.  This should be moved right above, or change the input type of the function.

**Pull request merged**